discovery/kubernetes/pod.go

```diff
@@ -213,7 +214,10 @@ func (p *Pod) buildPod(pod *apiv1.Pod) *targetgroup.Group {
 	tg.Labels = podLabels(pod)
 	tg.Labels[namespaceLabel] = lv(pod.Namespace)
 
-	for _, c := range pod.Spec.Containers {
+	containers := append(pod.Spec.Containers, pod.Spec.InitContainers...)
+	for i, c := range containers {
+		is_init := i >= len(pod.Spec.Containers)
```

Comment:
(nit) s/is_init/isInit/ to follow Go's [conventions](https://golang.org/doc/effective_go.html#mixed-caps).